<ui version="4.0" >
 <class>DiscoItemsWindowClass</class>
 <widget class="QMainWindow" name="DiscoItemsWindowClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>404</height>
   </rect>
  </property>
  <property name="iconSize" >
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="toolButtonStyle" >
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>3</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="lblJid" >
        <property name="text" >
         <string>Item:</string>
        </property>
        <property name="buddy" >
         <cstring>cmbJid</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbJid" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable" >
         <bool>true</bool>
        </property>
        <property name="insertPolicy" >
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblNode" >
        <property name="text" >
         <string>Node:</string>
        </property>
        <property name="buddy" >
         <cstring>cmbNode</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbNode" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable" >
         <bool>true</bool>
        </property>
        <property name="insertPolicy" >
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblSearch" >
        <property name="text" >
         <string>Search:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lneSearch" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QGroupBox" name="grbActions" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title" >
         <string>Actions</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignHCenter</set>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="trvItems" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>7</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="editTriggers" >
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="textElideMode" >
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="indentation" >
         <number>12</number>
        </property>
        <property name="allColumnsShowFocus" >
         <bool>true</bool>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="tlbToolBar" >
   <property name="movable" >
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
